src/main/resources/swagger/api-docs.yaml

```diff
@@ -4265,6 +4283,48 @@ definitions:
         type: string
         description: The translated use restriction of the consent.
 
+  structuredDataRequest:
+    type: object
+    properties:
+      generalResearchUse:
+        type: boolean
+        description: Is the data available for future general research use?
+      healthMedicalUseOnly:
+        type: boolean
+        description: Is future use limited for health/medical/biomedical research?
+      diseaseUseOnly:
+        items:
+          type: integer
+        description: Future use is limited to research involving the following disease area(s)
+      commercialUseProhibited:
+        type: boolean
+        description: Is future commercial use prohibited?
+      forProfitUseProhibited:
+        type: boolean
+        description: Is future use by for-profit entities prohibited?
+      methodsResearchProhibited:
+        type: boolean
+        description: Is future use for methods research (analytic/software/technology development) outside the bounds of the other specified restrictions prohibited?
+      aggregateLevelDataProhibited:
+        type: boolean
+        description: Is future use of aggregate-level data for general research purposes prohibited?
+      controlsUseProhibited:
+        type: boolean
+        description: Is future use as a control set for diseases other than those specified prohibited?
+      genderUseOnly:
+        type: string
+        description: If future use is limited to research involving a particular gender, specify "male" or "female". Any other value will be read as false.
+      pediatricResearchOnly:
+        type: boolean
+        description: Is future use limited to pediatric research?
+      IRB:
+        type: boolean
+        description: Should the requester provide documentation of local IRB/REC approval?
+      prefix:
+        type: string
+        description: A prefix that is prepended to the field names of the responses. This is purely for the user's personalization.
```

Comment:
Currently a prefix is required - should it be optional instead?

Reply:
Optional would make sense to me. I'm sure we'll see people passing in `""` if we make it required.